sklearn/feature_selection/_from_model.py

```diff
@@ -387,7 +408,7 @@ def threshold_(self):
         # SelectFromModel.estimator is not validated yet
         prefer_skip_nested_validation=False
     )
-    def partial_fit(self, X, y=None, **fit_params):
+    def partial_fit(self, X, y=None, **params):
```

Comment:
are we making the call consistent by calling it `**params` instead of `**fit_params`?

Reply:
In my opinion, it might avoid confusion to call those params like the function they are routing from (`partial_fit_params`, in this case). Imagining several sets of params being routed between several methods, this would help to read the code more easily. 

I'm not sure if Adrin shares the same sentiment about the term 'partial_fit_params,' @adrinjalali?

For me, the more explicit, the better.

Reply:
For me `fit_params` or `partial_fit_params` makes sense when those parameters are only dispatched to the `fit` or `partial_fit_params` from the underlying learners. If there is a scorer for instance, I am fine with `params`.

This is a bit the reason that I was a bit surprised here because we only pass it to the `partial_fit_params` of the sub-estimator.

Reply:
I personally don't care enough about these names really, users can't pass them explicitly anyway.

Reply:
So let's be intuitive and consistent for the sake of the documentation :).